base/statistics.jl

```diff
@@ -177,29 +241,48 @@ _median(v::AbstractArray{T}, ::Colon) where {T} = median!(copyto!(Array{T,1}(und
 # for now, use the R/S definition of quantile; may want variants later
 # see ?quantile in R -- this is type 7
 """
-    quantile!([q, ] v, p; sorted=false)
+    quantile!([q::AbstractArray, ] v::AbstractVector, p; sorted=false)
 
-Compute the quantile(s) of a vector `v` at the probability or probabilities `p`, which
-can be given as a single value, a vector, or a tuple. If `p` is a vector, an optional
+Compute the quantile(s) of vector `v` at a specified probability or vector or tuple of
```

Comment:
I think you'd still want *a* or *the* vector and I'm not sure the reformulation is an improvement.

Reply:
OK, I've just tried to use the same wording for `quantile` and `quantile!`. Can you suggest something else?

Reply:
The comment included two proposals: either keep the old formulation as it is or just add "a" or "the" before "vector".